tests/tests_lite/helpers/models.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Iterator
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+from torch.nn import Module
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader, Dataset, IterableDataset
+
+from lightning_lite import LightningLite
+
+
+class RandomDataset(Dataset):
```

Comment:
@carmocca @akihironitta @Borda is this something we might "offload" to the utils package so that we don't have a duplication of this in PL and Lite?

Reply:
No because this requires PyTorch to be installed.